tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -674,6 +769,8 @@ def make_random_operation(self) -> Operation:
             lambda: UpdateConfigOperation(),
             RedpandaAdminOperation.PRODUCE_TO_TOPIC:
             lambda: ProduceToTopic(self.prefix),
+            RedpandaAdminOperation.DELETE_RECORDS:
+            lambda: DeleteRecords(self.prefix),
```

Comment:
nit: at some point it'd be great if we had this fuzzer consume and expect specific error codes or number of results